docs/apis/streaming/connectors/kinesis.md

```diff
@@ -240,8 +239,9 @@ consumer when calling this API can also be modified by using the other keys pref
 ### Kinesis Producer
 
 The `FlinkKinesisProducer` is used for putting data from a Flink stream into a Kinesis stream. Note that the producer is not participating in
-Flink's checkpointing and doesn't provide exactly-once processing guarantees. In case of a failure, data will be written again
-to Kinesis, leading to duplicates. This behavior is usually called "at-least-once" semantics.
+Flink's checkpointing and doesn't provide exactly-once processing guarantees. 
+Also, the Kinesis producer does not guarantee that records are written in order to the shards (See [here](https://github.com/awslabs/amazon-kinesis-producer/issues/23) and [here](http://docs.aws.amazon.com/kinesis/latest/APIReference/API_PutRecord.html#API_PutRecord_RequestSyntax) for more details). 
+In case of a failure, data will be written again to Kinesis, leading to duplicates. This behavior is usually called "at-least-once" semantics.
```

Comment:
Should we also mention that resharding streams will using the Kinesis Producer will also result in duplicates? I experienced this while testing exactly-once with resharding before.